week16-18-05-18.md

```diff
@@ -0,0 +1,65 @@
+# Week 16
+
+## News
+
+## New Features
+
+## NIO
+
+## Community Contributions
+
+## Tagged releases
+- [Crypto 3.1.7](https://github.com/vapor/crypto/releases/tag/3.1.7)
+- [PostgreSQL 1.0.0 RC 2.2.1](https://github.com/vapor/postgresql/releases/tag/1.0.0-rc.2.2.1)
+- [HTTP 3.0.6](https://github.com/vapor/http/releases/tag/3.0.6)
+- [Auth 2.0.0 RC 4.1](https://github.com/vapor/auth/releases/tag/2.0.0-rc.4.1)
+
+## Articles
+
+### Username/Password Authentication in Vapor 3
+Brought to you by [@anapaix](https://github.com/JoeyBodnar)
+
+[Username/Password Authentication with Vapor 3](https://www.vaporforums.io/thread/43)
+
+### Token Authentication in Vapor 3
+Brought to you by  [@anapaix](https://github.com/JoeyBodnar)
+
+[Token Authentication with Vapor 3](https://www.vaporforums.io/thread/44)
+
+### How to get started with The Vapor 3 Framework
+Brought to you by [@steffendsommer](https://github.com/steffendsommer) and [@siemensikkema](https://github.com/siemensikkema)
+
+[Get started with Vapor 3](https://www.nodesagency.com/how-to-get-started-with-the-vapor-3-framework/)
+
+### Playing Pokemon with Vapor
+Brought to you by [@sagnew](https://github.com/sagnew)
```

Comment:
You'll want to note that the tutorial is for Vapor 2

Reply:
Updated. Thank you very much.